PayloadPkg/FirmwareUpdate/FirmwareUpdate.c

```diff
@@ -975,6 +974,7 @@ ProcessCapsule (
     FwUpdStatus.Version = FW_UPDATE_STATUS_VERSION;
     FwUpdStatus.Length = sizeof(FW_UPDATE_STATUS);
     FwUpdStatus.RetryCount = InitFwuRetryCount ();
+    FwUpdStatus.CsmeNeedReset = 0xF;
```

Comment:
how about giving it the value (0xf) a name. And why not 0xff?

Reply:
Because it is of size UINT8. And you are right if adding more comments to describe the meaning of the magic number.

Reply:
Just because it is UINT8, why use 0xF here?